Formula/pueue.rb

```diff
@@ -72,7 +73,10 @@ def plist
       Process.kill("TERM", pid)
     end
 
-    assert_match "Pueue daemon #{version}", shell_output("#{bin}/pueued --version")
-    assert_match "Pueue client #{version}", shell_output("#{bin}/pueue --version")
+    # bug report about the version in the release artifact
+    # hopefully revert the change in next release
+    # https://github.com/Nukesor/pueue/issues/169
```

Comment:
Can't we get a real test instead?

Reply:
A version with a fix has been released: https://github.com/Nukesor/pueue/releases/tag/pueue-v0.11.1, but we definitely need a better test in addition to this one.

Reply:
I will give a shot.

Reply:
I think it should be good, let me know your thought.